cmake/ProjTest.cmake

```diff
@@ -14,20 +14,28 @@ function(proj_test_set_properties TESTNAME)
     PROPERTY ENVIRONMENT ${_env})
 endfunction()
 
-function(proj_add_test_script_sh SH_NAME BIN_USE)
-  if(UNIX)
```

Comment:
this `if(UNIX)` check was moved to where the function as used to make it more clear that these tests are skipped for Windows.